qa/tasks/cephfs/test_dump_tree.py

```diff
@@ -39,7 +39,7 @@ def test_random(self):
 
         self.populate()
         inos = self.get_paths_to_ino()
-        target = random.choice(inos.keys())
+        target = random.sample(inos.keys(), 1)[0]
```

Comment:
same here (shouldn't you use `iter` here to access the first element?)

Reply:
ditto.